### PR TITLE
Frontend improvements to the viewall students page.

### PR DESCRIPTION
**Pull request by aniarya82**

### CODE REVIEW

File: app/views/viewall/students.html.erb [hunk 1]
```diff
@@ -1,6 +1,39 @@
 <h2 align="center">Complete Student Member Status</h2>
 <hr>
 
+<style type="text/css">
+.label {
```
Comment (icyflame) [inline app/views/viewall/students.html.erb]: @aniarya82

Why is this necessary? I think a screenshot will better explain this, if it has any yo improvements.

Comment (icyflame) [inline app/views/viewall/students.html.erb]: @aniarya82 

Why is this necessary?

I think a screenshot will be better if this improves the front end of the site.

**Comment by aniarya82** [comment]
screen shot 2015-06-17 at 7 48 00 pm

**Comment by icyflame** [comment]
Great! The UI looks better now! Kudos! :+1:

I will do a merge at my end, and push the changes soon.

**Comment by aniarya82** [comment]
you satisfied with the query optimization ..??

**Comment by icyflame** [comment]
Yes, I think so. I didn't know that `where` can be used on ActiveRecord Relations! But it makes sense, right! Actually, the whole processing can be done on server side, instead of DB side. 

Of course, I have to check if there is more than one query being issued from the site. 

Have you checked that on the local host server? (The number of queries sent when that particular page is loaded)

**Comment by aniarya82** [comment]
i don't have idea how to do that :/

**Comment by icyflame** [comment]
@aniarya82 
It might be time to learn how to check the server logs, for errors and queries sent.

File: app/views/viewall/students.html.erb [hunk 2]
```diff
@@ -22,10 +55,12 @@
 		<% @neutral = 0 %>
 		<% @registered = 0 %>
 		<tr>
-			<% allties = TieAlumniWithStudentMember.where({ :studentmember_id => "#{student.id}" }) %>
+			<%# allties = TieAlumniWithStudentMember.where({ :studentmember_id => "#{student.id}" }) %>
+			<% allties = @allTies.where({ :studentmember_id => "#{student.id}" }) %>
```
Comment (icyflame) [inline app/views/viewall/students.html.erb]: @aniarya82 

unfortunately, there is no optimization here. The `where` is interpreted as a query, and there are still the same number of queries. (one query per alumni, and one query per student member, comes to a total of around 1200 queries for one page load, in production).

You will have to find some other way.

**Comment by icyflame** [comment]
@aniarya82 

While you figure this optimization out, I am going to merge the frontend changes that you have made in this pull request. Please open a new PR when you are done with the optimization.

**Comment by icyflame** [comment]
@aniarya82 

I have merged these changes in without any edits.
